mods/ra/maps/evacuation/evacuation.lua

```diff
@@ -0,0 +1,365 @@
+DeathThreshold =
+{
+	Easy = 200,
+	Normal = 100,
+}
+
+TanyaType = "e7"
+TanyaStance = "AttackAnything"
+if Map.Difficulty ~= "Easy" then
+	TanyaType = "e7.noautotarget"
+	TanyaStance = "HoldFire"
+end
+
+RepairTriggerThreshold =
+{
+	Easy = 50,
+	Normal = 75
+}
+
+Sams = { Sam1, Sam2, Sam3, Sam4 }
+TownUnits =
+{
+	Einstein, Engineer,
+	TownUnit01, TownUnit02, TownUnit03, TownUnit04, TownUnit05, TownUnit06, TownUnit07,
+	TownUnit08, TownUnit09, TownUnit10, TownUnit11, TownUnit12, TownUnit13, TownUnit14
+}
+
+ParabombDelay = DateTime.Seconds(30) 
+ParatroopersDelay = DateTime.Minutes(5)
+Paratroopers =
+{
+	{
+		-- `proxy` cannot be assigned in global scope since a player is needed to create the actor
+		proxy = nil,
+		entry = BadgerEntryPoint1.Location,
+		drop  = BadgerDropPoint1.Location,
+	},
+	{
+		proxy = nil,
+		entry = BadgerEntryPoint1.Location + CVec.New(3, 0),
+		drop  = BadgerDropPoint2.Location,
+	},
+	{
+		proxy = nil,
+		entry = BadgerEntryPoint1.Location + CVec.New(6, 0),
+		drop  = BadgerDropPoint3.Location,
+	},
+}
+
+AttackGroup = { }
+AttackGroupSize = 5
+SovietInfantry = { "e1", "e2", "e3" }
+SovietVehiclesUpgradeDelay = DateTime.Minutes(4)
+SovietVehicleType = "Normal"
+SovietVehicles =
+{
+	Normal = { "3tnk" },
+	Upgraded = { "3tnk", "v2rl" },
+}
+ProductionInterval =
+{
+	Easy = DateTime.Seconds(10),
+	Normal = DateTime.Seconds(2)
+}
+
+ReinforcementsDelay = DateTime.Minutes(16)
+ReinforcementsUnits = { "2tnk", "2tnk", "2tnk", "2tnk", "2tnk", "2tnk", "1tnk", "1tnk", "jeep", "e1",
+	"e1", "e1", "e1", "e3", "e3", "mcv", "truk", "truk", "truk", "truk", "truk", "truk" }
+
+SpawnAlliedReinforcements = function()
+	if allies2.IsLocalPlayer then
+		UserInterface.SetMissionText("")
+		Media.PlaySpeechNotification(allies2, "AlliedReinforcementsArrived")
+	end
+	Reinforcements.Reinforce(allies2, ReinforcementsUnits, { ReinforcementsEntryPoint.Location, Allies2BasePoint.Location })
+end
+
+Yak = nil
+YakAttack = function(yak)
+	local targets = Map.ActorsInCircle(YakAttackPoint.CenterPosition, WDist.FromCells(10), function(a)
+		return a.Owner == allies1 and not a.IsDead and a ~= Einstein and a ~= Tanya and a ~= Engineer
+	end)
+
+	if (#targets > 0) then
+		yak.Attack(Utils.Random(targets))
+	end
+	yak.Move(Map.ClosestEdgeCell(yak.Location))
+	yak.Destroy()
+	Trigger.OnRemovedFromWorld(Yak, function()
+		Yak = nil
```

Comment:
Hm, I don't think we really need this, we could just add an `Yak.IsInWorld` check at the `OnEnteredProximityTrigger` instead: `if Yak.IsInWorld or actor.Owner ~= allies1 `